src/main/java/seedu/address/logic/parser/MatchCommandParser.java

```diff
@@ -19,14 +21,19 @@
     public MatchCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args);
+        String preamble = argMultimap.getPreamble().trim();
+        // Check if argument passed is a number or not
+        if (!StringUtil.isNumeric(preamble)) {
```

Comment:
Shouldn't you check for integer instead of numeric?

Reply:
Changed as per our discussion.